CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ option(ONLY_INSTALL "Only install" OFF)
 # CMake modules
 list(APPEND CMAKE_MODULE_PATH
   ${CMAKE_CURRENT_SOURCE_DIR}/cmake
-  ${HIP_PATH}/lib/cmake/hip ${HIP_PATH}/cmake /opt/rocm/lib/cmake/hip /opt/rocm/hip/cmake # FindHIP.cmake
+  ${ROCM_PATH}/lib/cmake/hip ${HIP_PATH}/cmake /opt/rocm/lib/cmake/hip /opt/rocm/hip/cmake # FindHIP.cmake
```

Comment:
Does `${HIP_PATH}/cmake` also need to be changed? If not, LGTM.

Reply:
No, As this was existing search path, (like /opt/rocm/hip/cmake).  Thanks